velox/dwio/common/DirectBufferedInput.cpp

```diff
@@ -148,7 +141,7 @@ void DirectBufferedInput::makeLoads(
       requests,
       maxDistance,
       // Break batches up. Better load more short ones i parallel.
-      1000, // limit coalesce by size, not count.
+      std::numeric_limits<int32_t>::max(), // limit coalesce by size, not count.
```

Comment:
We want to keep cache as small as 40? Thanks!

Reply:
No I checked with Orri and there is no reason we must keep it 40, I will change that